Add Report

hoco1 opened the pull request:
Hey bro 
I created report with power bi  and add into readme files
if you like it, merge it

hoco1 committed:
Update README.md

hoco1 committed:
Update README.md

hoco1 committed:
Update README.md

arshin1989 commented:
> Hey bro
> I created report with power bi and add into readme files
> if you like it, merge it

Hi Hoco,
It's great to have this data on a dashboard, and thank you so much for taking the time to do this.
Just one point, the link you provided did not open for me, and I encounter this error message:
The link you tried to open is not available to you or has been removed.
Are you sure the link is correct?
Thanks

hoco1 committed:
add dashboard imgs

hoco1 committed:
modify Readmes

hoco1 committed:
add .pbix in directory

hoco1 committed:
Update README.md

hoco1 committed:
Update README.md

hoco1 commented:
Yes bro you say correct 
now I fixed it 
but one problem appear if you want to see online you must be sign up then you can see 
if you don't like to sign up, I added "temp.pbix" you can see it offline

arshin1989 commented:
> Yes bro you say correct
> now I fixed it
> but one problem appear if you want to see online you must be sign up then you can see
> if you don't like to sign up, I added "temp.pbix" you can see it offline

Your work is great and thank you very much again for your contribution.
